types/ldapjs-client/index.d.ts

```diff
@@ -0,0 +1,92 @@
+// Type definitions for ldapjs-client 0.1
+// Project: https://github.com/zont/ldapjs-client#readme
+// Definitions by: Valerio Coltrè <https://github.com/colthreepv>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 3.0
+
+declare class Filter {
+    matches(obj: any): boolean;
+    type: string;
+}
+
+interface Change {
+    operation: string;
+    modification: {
+        [key: string]: any;
+    };
+}
+
+interface SearchOptions {
```

Comment:
If this is only used in one place, I'd really just avoid writing two types.

Reply:
sure